### ISSUE TITLE
Badger News email - Alt text needed on a few elements

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#433]
**Description of the issue:**
We are currently missing appropriate alt text for a few elements within the Badger News email.

They are:

- The Red Badger logo
- The "Catch up on the talks" video still
- The "React London 2017 vlog" video still

As it currently is, these elements make no sense to someone using a screenreader

**Browser used:**
All

**OS used:**
All

**Device used:**
All

**Steps to reproduce:**

1. Turn on Voice Assist on Mac
2. Open the Badger News email in a browser
3. Navigate through the email

**Actual result:**
The three elements listed above contain no alt-text

**Expected result:**
A user should be able to navigate the email with a screenreader and make sense of all content

**Comment by mveritym** [comment]
@gregdorward Here is the new campaign with alt text for those images added: https://us6.admin.mailchimp.com/campaigns/wizard/neapolitan?id=1235273

**Comment by gregdorward** [comment]
great, thanks!

**Comment by gregdorward** [comment]
This one is fixed and can now be closed
